src/interface.jl

```diff
@@ -302,7 +314,11 @@ function solve(
         num_int,
     )
 
-    fw_callback = build_FW_callback(tree, min_number_lower, true, fw_iterations, min_fw_iterations, time_ref, tree.root.options[:time_limit])
+    if tree.root.options[:variant] != DICG()
+        fw_callback = build_FW_callback(tree, min_number_lower, true, fw_iterations, min_fw_iterations, time_ref, tree.root.options[:time_limit])
+    else
+        fw_callback = build_FW_callback(tree, min_number_lower, true, fw_iterations, min_fw_iterations, time_ref, tree.root.options[:time_limit]; use_DICG=true)
+    end
```

Comment:
To make to shorter,
`fw_callback = build_FW_callback(tree, min_number_lower, true, fw_iterations, min_fw_iterations, time_ref, tree.root.options[:time_limit], use_DICG=tree.root.options[:varinat] == DICG())`